transcendence-app/src/app.module.ts

```diff
@@ -11,6 +14,16 @@ import { validate } from './config/env.validation';
       validate,
     }),
     OAuth42Module,
+    TypeOrmModule.forRoot({
+      type: 'postgres',
+      host: 'db',
+      port: 5432,
+      username: 'transcendence',
+      password: 'example',
```

Comment:
suggestion:  We could use the `ConfigService` here and add this info to a .env file or environment variables.

Reply:
Yes, good point. If we finally decide to use TypeORM we can do that or even we can call a config file to clean the app.module with `TypeOrmModule.forRoot(config.typeorm.ts)` and in that file add the configuration with environment variables as you mentioned.